portal-web/test/functional/com/liferay/portalweb/tests/MyDMTests/MyDMTests.testcase

```diff
@@ -0,0 +1,64 @@
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Can Create a Documents and Media"
+	@priority = "5"
+	test CanCreateDM {
+		
+        task("Create new page and widget media"){
+        JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Documents and Media Page");
+		JSONLayout.addWidgetToPublicLayout(
+			column = "1",
+			groupName = "Guest",
+			layoutName = "Documents and Media Page",
+			widgetName = "Documents and Media");
+        }
+
+         task("Add new media file"){
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Documents and Media");
+
+		DMDocument.addCP(
+			dmDocumentDescription = "Document Description",
+			dmDocumentFile = "Document_2.jpg",
+			dmDocumentTitle = "DM Test");
+        }
+
+		ProductMenu.gotoHomePage();
+
+        task("Download media"){
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Documents and Media");
+
+		DMNavigator.openToEntryInAdmin(
+			dmDocumentTitle = "DM Test",
+			groupName = "Guest",
+			siteURLKey = "guest");
+
+		Button.click(key_text = "Download");
+        }
```

Comment:
Oi @AndersonGuima, você está fazendo o teste de criar o DM e verificar se ele está presente, então não precisa fazer download, poderia modificar isso?

Você deve verificar se o DM criado está presente na página que você criou (Documents and Media Page)

Reply:
Conseguir enviar com o nome certo, agora vou tentar concertar isso que voce disse. Valeu